### ISSUE TITLE
What happens when an infected person is recovering or recovers?

### ISSUE THREAD

**Issue by heroichornet** [DP-3T/documents#199]
- A person is infected, tests positive.
- Code is entered by medical professional.
- Persons symptoms are mild, person is told to stay at home.

Maybe unlikely at the moment as only severe cases are tested, but exceptions exist when: 
_Testing capacity is high relative to infected people._
_Person works in health care_

What next?
* Do people continue to be informed in case of contact?
* Assuming you can be re-infected, will the status of the person be reset by a medical professional also?
* A person with mild symptoms still may be forced to go outside to go to a pharmacy, buy groceries, get fresh air. Contact may still occur. Does the app report this? Maybe they wear facemasks, maybe not?

I did not see any mention in the documentation and I also didn't see any guiding information in the alpha android app during testing, when I registered myself as infected - maybe I've missed something.

**Comment by lbarman** [comment]
Hi @heroichornet; thanks for your input. For privacy reasons, once an upload is done, this person changes her identity (secret seeds). With respect to the system, she becomes a completely new person that is "not infected" anymore.

I think part of this answer relies on the duration of the infectious period, and I'll have to double-check to say something accurate. Ping me again if I forget!

Thanks

**Comment by heroichornet** [comment]
> With respect to the system, she becomes a completely new person that is "not infected" anymore.

This is not clearly communicated by the app, if you look at the screenshot. Will an infected person that continues to use the app be informed if they come in contact with an infected individual? UI doesn't make this  clear

![image](https://user-images.githubusercontent.com/1138612/79685192-ff391f80-8236-11ea-8e05-17782a3aa6ef.png)

> this is out-of-scope. The goal of our system is to notify at-risk users, after that it's up to the health authorities.

If this is the case then having  active bluetooth does not benefit the system or the user once infected, yet my app continues to run in the background after infection. I think the app should either:

* disable bluetooth 
* Allow the user to reset the app to initial state, so he may use it again in normal operation

**Comment by lbarman** [comment]
Hi @heroichornet; thanks for the input! Sorry for the ping-pong but could you report this on the [app's repo](https://github.com/DP-3T/dp3t-app-ios) ? this one focuses on the design of the system.
Thanks!
